great-idea-website/css/index.css

```diff
@@ -63,4 +63,118 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Your code starts here! */
\ No newline at end of file
+/* Your code starts here! */
+
+.container {
+  width: 80%;
+  min-width: 600px;
+  max-width: 1000px;
+  margin: 0 auto;
+  align-content: center;
+}
+.navbar{
+  margin: 0 auto;
+  justify-content: space-evenly;
+  word-spacing: 50px;
+  display:inline-block;
+  position: relative;
+  top: 1em;
+
+}
+.navbar nav a{
+	font-size: 10px;  
+	color: grey;
+	border: 1px transparent solid;
+	width: 80%;
+	height: 20%;
+	margin: 0 auto;	
+	padding-top: 50px;
+}
+.navbar .image1{
+  position: absolute;
+  right: -20em;
+  top: -.5em;
+ }
+.cont .slg{
+	font-family: 'Bangers' cursive;
+	font-size: 55px;
+	text-align: center;
+	width: 500px;
+	display: inline-block;
+	margin-bottom: 100px;
+	vertical-align: center;
+	
+}
+.cont .slg button{
+	font-size: 12px;
+	padding: 6px 28px;
+	background: transparent;
+	border: 1px black solid;
+}
+.cont .image2{
+	display: inline-block;
+	margin-top: 100px;
+	margin-bottom: -100px;
+	border-radius: 90px;
+
+}
+
+.paragraphs1{
+	display: inline-block;
+	
+}
+.par1{
+	display: inline-block;
+	border: 1px transparent solid;
+	width: 40%;
+	height: 40%;
+}
+.par2{
+	display: inline-block;
+	border: 1px transparent solid;
+	width: 40%;
+	height: 20%;
+}
+.middle-image{
+	height: 20%;	
+	align-content: center;
+	margin-top: 50px;
+}
+.paragraphs2{
+	display: inline-block;
```

Comment:
you can have a border-bottom: 1px solid black here too for the bottom border.  Also add margin